apps/grafana/vars/all/main.yml

```diff
@@ -0,0 +1,50 @@
+# Application default configuration
+
+# -- route
+grafana_host: "grafana.{{ namespace_name }}.{{ domain_name }}"
+
+# -- databases (defaults to postgresql)
+grafana_postgresql_version: "13.3"
+grafana_database_engine: "postgres"
+grafana_database_image_name: "postgres"
+grafana_database_image_tag: "13.3"
+grafana_database_host: "postgresql"
+grafana_database_port: 5432
+grafana_database_secret_name: "grafana-database-{{ grafana_vault_checksum | default('undefined_grafana_vault_checksum') }}"
+
+# -- grafana
+grafana_image_name: "grafana/grafana"
+grafana_image_tag: "8.0.3"
+grafana_container_uid: 472
+grafana_container_gid: 0
+# The image pull secret name should match the name of your secret created to
+# login to your private docker registry
+grafana_image_pull_secret_name: ""
```

Comment:
Not sure to understand, where is this match checked?

Reply:
It is used here in the deployment: https://github.com/openfun/arnold-apps/pull/8/files#diff-deb9262786842ff19dea29f66db2f2418415d9b446008a24967a5bdd4d8e89c2R42